community/STEERING-COMMITTEE.md

```diff
@@ -1,61 +1,123 @@
 # Knative Steering Committee
 
-The Knative Steering Committee (SC) defines, evolves, and defends the vision,
-values, mission, and scope of the project. _The Steering Committee is a
-work-in-progress._
+The Knative Steering Committee (KSC) is the ultimate authority for the Knative
+project, and controls all aspects of the project or delegates its authority
+to other committees.
+
+KSC has delegated technical guidance for the project to the
+[Technical Oversight Committee](TECH-OVERSIGHT-COMMITTEE.md). However, the KSC
+acts as the final escalation point for any and all decisions within the project.
```

Comment:
Since this is cover below in the Charter section https://github.com/knative/docs/pull/1017/files#diff-f02a39b268435b05d07ea4fabe284e16R34

I recommend removing the sentence _However, the KSC
acts as the final escalation point for any and all decisions within the project._